ci/conda/environments/dev_env.yml

```diff
@@ -1,70 +0,0 @@
-# SPDX-FileCopyrightText: Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# Dependencies needed for development environment. Runtime deps are in meta.yml
-name: mrc
-channels:
-  - rapidsai
-  - nvidia/label/cuda-11.8.0
-  - nvidia
-  - rapidsai-nightly
-  - conda-forge
-dependencies:
-  - autoconf>=2.69
-  - bash-completion
-  - benchmark=1.6.0
-  - boost-cpp=1.82
-  - ccache
-  - cmake=3.25
-  - cuda-toolkit # Version comes from the channel above
-  - cxx-compiler # Sets up the distro versions of our compilers
-  - doxygen=1.9.2
-  - flake8
-  - flatbuffers=2.0
-  - gcovr=5.0
-  - gdb
-  - gflags=2.2
-  - git>=2.35.3 # Needed for wildcards on safe.directory
-  - glog=0.6
-  - gmock=1.13
-  - graphviz=3.0
-  - libgrpc=1.54.0
-  - gtest=1.13
-  - gxx=11.2 # Specifies which versions of GXX and GCC to use
-  - isort
-  - jinja2=3.0
-  - lcov=1.15
-  - libhwloc=2.9.2
-  - libprotobuf=3.21
-  - librmm=23.06
-  - libtool
-  - ninja=1.10
-  - nlohmann_json=3.9
-  - numactl-libs-cos7-x86_64
```

Comment:
To get this to build on my system, I needed to add `numactl-libs-cos7-x86_64` to the build dependencies. Looks like this package didnt make the migration to `dependencies.yml`